### ISSUE TITLE
Found something which causes untrackable errors!

### ISSUE THREAD

**Issue by nealoke** [lkytal/quickTask#19]
Hi! I've just have had a couple of days where I experienced an error which I could not find from the error messages. OMG i'm so happy I found it :p and it has to do with this plugin 👍 

Currently when I fire a `task` from `quickTask` dropdown menu, it runs the following: `cd /d "c:\xampp\htdocs\wizer\packages\website"`. To be honest it works so fine, however when running my webpack config it causes all sorts of weird errors such as:

- `Module did not self-register`
- `Module parse failed: Unexpected token (105:3) You may need an appropriate loader to handle this file type.`

```
There are multiple modules with names that only differ in casing.
This can lead to unexpected behavior when compiling on a filesystem with other case-semantic.
Use equal casing. Compare these module identifiers:
* C:\xampp\htdocs\wizer\packages\dashboard\node_modules\process\browser.js
```

The issue is a classic in programming, namely a letter that is normally `UPPERCASE` being used in `lowercase`. Which is not obvious...

So when I manually do the `cd packages/website` it goes to `C:\xampp\htdocs\wizer\packages\website`, notice that `C` is in capital letter. When running the same command as I would via `quickTask` it runs fine, no errors anywhere.

**Solve this by making sure that the letter of the volume is capitalized**. This took me a total of 3 days to figure out what was going wrong. I tried removing all node_modules, updating most libraries, etc. I hope this helps anybody else who is on windows and trying to figure out what the heck is wrong.

**Comment by lkytal** [comment]
Thank yours for your feedback, an impressive finding of such a subtle problem. I've published 3.1.3 to address this problem.

**Comment by nealoke** [comment]
@lkytal I just hope nobody else is putting in a lot of time to find this issue :D and it was a 💥 in th *** :D Thanks for fixing it!
